### DISCUSSION TITLE
What are the ways to access the audio controls?

### DISCUSSION THREAD

**Discussion by mowestusa** [rern/rAudio#227] [General]
I have a Raspberry 1 B that I have RuneAudio installed on and running. I can only connect to it successfully using a Firefox on a Linux desktop or on a Windows 10 laptop on the same network using the address = `http://runeaudio.local` The RuneAudio app on Android does not work, nor does Firefox on Android work. Because of other "broken" functionality, I was hoping to give rAudio-1 a try.

I'm planning to run the Raspberry 1 B headless like it is right now. What are the different ways that I can access the rAudio-1 interface?

- I thought I read that it can be accessed by going to = `http://raudio.local` so I'm assuming that this would work the same way that I have it working on the Linux desktop and Windows 10 laptop in Firefox. Is there a way to access rAudio-1 using an Android phone or other ways that I haven't found in the discussions and Readme?

**Comment by mrManniManfred** [comment]
You can look up the IP of your Raspberry in your router.

So instead accessing it via http://runeaudio.local use something like e.g. 192.168.0.3 depending on your DHCP..

**Reply by mowestusa** [reply]
That did work, thanks for the suggestion. I'll assume that I will need to do that with rAudio-1's interface as well in order to access it on a phone.

**Reply by mrManniManfred** [reply]
Yes I think it is because of availability or lack of [Avahi](https://en.wikipedia.org/wiki/Avahi_(software)) 

compare also: 
https://en.wikipedia.org/wiki/Zero-configuration_networking
https://en.wikipedia.org/wiki/.local
